src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -80,14 +82,19 @@
      * Opens the given Inventory for the player to view.
      *
      * @param inventory The inventory to view
+     * @param cause The {@link Cause} to use when opening the inventory
+     * @throws IllegalArgumentException if a {@link PluginContainer} is not the root of the cause
```

Comment:
Seems reasonable. We require it on cause-overload-methods in `Extent` so this is basically the same